src/App.js

```diff
@@ -1,17 +1,34 @@
 import React from 'react';
 import './App.scss';
 
+import TodoList from './components/TodoList/TodoList';
+
 import todos from './api/todos';
 import users from './api/users';
 
+const preparedTodos = todos.map((todo) => {
+  const todoCopy = { ...todo };
+
+  users.forEach((user) => {
+    if (user.id === todo.userId) {
+      todoCopy.user = user;
+    }
+  });
```

Comment:
Тут можно сократить немного кода, если в `todoCopy` не только развернуть исходный `todo`, но и сразу присвоить ключу `user` значение в которое положить отфильтрованного методом `filter` или найденного методом  `find` искомого юзера. 
```suggestion
  const todoCopy = { 
     ...todo,
     user: users.find(({ id }) => id === todo.userId),
   };
```